### PR TITLE
[HDRP Template] Upgrade Input System to 1.8.1 in 3D Sample

### PR DESCRIPTION
**Pull request by jfreire-unity**
---
### Purpose of this PR

- Upgrading to latest Input System package 1.8.1
- Changing Project Settings > Active Input Handling to "Both". This should overcome issues with UGUI reported [here](https://jira.unity3d.com/browse/UUM-68200).

---
### Testing status
❌ No local testing done yet

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Review by lyndon-unity — CHANGES_REQUESTED** [review]
Need to add EditorBuildAsset file that references InputSystem_Actions.asset

Currently this file has no reference to the newly created project wide input actions asset in the Assets folder
https://github.cds.internal.unity3d.com/unity/com.unity.template.urp-sample/blob/1997b98e6d98982ac2b7bbe59494d2a631491f84/ProjectSettings/EditorBuildSettings.asset#L26

File: com.unity.template-hd/ProjectSettings/EditorBuildSettings.asset [hunk 1]
```diff
@@ -6,6 +6,8 @@ EditorBuildSettings:
   serializedVersion: 2
   m_Scenes:
   - enabled: 1
-    path: Assets/SampleSceneAssets/Scenes/SampleScene.unity
+    path: Assets/Scenes/SampleScene.unity
```
Comment (lyndon-unity) [inline com.unity.template-hd/ProjectSettings/EditorBuildSettings.asset:9]: This line change is probably unnecessary AFAICT - can you revert that one line?

**Review by lyndon-unity — CHANGES_REQUESTED** [review]
Config objects info in EditorBuildSettings looks good - but looks like one extra unnecessary line changes which is probably not needed and may have a side effect. Suggest manually putting that one line back

File: com.unity.template-hd/ProjectSettings/EditorBuildSettings.asset (hunk 1 above)
Comment (graham-huws) [reply com.unity.template-hd/ProjectSettings/EditorBuildSettings.asset:9]: Sure, can do - the (updated) path is correct though, the old one doesn't exist, is that fine?

**Review by Pauliusd01 — APPROVED** [review]
LGTM, the player is set to use both backends, EventSystem no longer throws the errors and a default ProjectWideActions asset is present in the main Assets directory (The only thing that might be a bit odd that It's just a loose file instead of being nested in some Settings folder)

Also, double checked basic uGUI and UIToolkit functionality by creating and checking functionality of buttons, toggles, input fields, etc.

File: com.unity.template-hd/ProjectSettings/EditorBuildSettings.asset (hunk 1 above)
Comment (lyndon-unity) [reply com.unity.template-hd/ProjectSettings/EditorBuildSettings.asset:9]: Ok if that's a correction then great lets land it as is. Thank you.

**Review by lyndon-unity — APPROVED** [review]

**Review by graham-huws — APPROVED** [review]

**Review by LagueKristin — APPROVED** [review]
Fixed outstanding issues, bumped version and changelog changes!

**Pull request merged**